test/javascript/src/config/selectors.test.js

```diff
@@ -47,48 +48,123 @@ describe("getAllActions", () => {
   })
 })
 
-describe("getSortedActionsForIndicator", () => {
-  describe("when there are no disease specific actions", () => {
-    const actionsToSort = [
-      { id: 1, level: 2, sequence: 2, disease_id: null },
-      { id: 2, level: 2, sequence: 1, disease_id: null },
-      { id: 3, level: 1, sequence: 2, disease_id: null },
-      { id: 4, level: 1, sequence: 1, disease_id: null },
+describe("getSortedActions", () => {
+  let technicalAreaMap, indicatorMap, actionsToSort, sortedActions
+
+  beforeEach(() => {
+    const state = store.getState()
+    const actionIdsToSort = [
+      // These ids are copied from the actions that are displayed when you go to Action Type tab
+      // and then the Assessment and Data Use bar. These ids will likely change when more Health security actions
```

Comment:
I find this unclear and difficult to understand, I'll take a shot at an overhaul of the wording, but its okay for now.